wsagent/che-core-api-languageserver/src/main/java/org/eclipse/che/api/languageserver/CheLanguageClient.java

```diff
@@ -84,4 +89,17 @@ public void logMessage(MessageParams message) {
         break;
     }
   }
+
+  @Override
+  public CompletableFuture<Void> registerCapability(RegistrationParams params) {
+    return CompletableFuture.completedFuture(null); // runAsync(
```

Comment:
I think we should either remove this comment or add proper LOG message.

Reply:
yes, thx. Will do